spec/controllers/apps_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'spec_helper'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe AppsController do
```

Comment:
@adamruzicka, shouldn't we reuse `spec/integration/apps_spec.rb`?

Reply:
I'm ok with having specs in `spec/controllers` be more about what the controllers do and specs in `spec/integration` more about describing the api with rswag. I could however be convinced otherwise

Reply:
In this case, we should move it to integration and update api description accordingly.

Reply:
Honestly, I overlooked `apps_spec.rb` as it didn't look like a controller spec from the filename.
Let me know where I should move the tests, I'm happy to move them to `apps_spec.rb`, as they feel more like they need to be API tests.

Reply:
`apps_spec.rb` is the right place for this